### PR TITLE
User event tutorials

### PR DESCRIPTION
**Pull request by pmichalskiGD**
* write user events tutorial added

### CODE REVIEW

**Review by DawidJeziorskiGrid — APPROVED** [review]

**Review by marcoles — APPROVED** [review]

File: tutorials/retail_api_v2_rejoin_user_events_java.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    The code samples for each of the Retail services are stored in different directories.
+
+1. Go to the code samples directory - our starting point to run more commands.
+    ```bash
+    cd java-retail/samples/snippets
+    ```
+
+## Rejoin user events
+
+1. Before you start, build the Maven project and go to the code samples directory - our starting point to runcode samples:
```
Comment (DawidJeziorskiGrid) [inline tutorials/retail_api_v2_rejoin_user_events_java.md:109]: Is:
our starting point to runcode samples:
maybe it should be:
our starting point to run code samples:

File: tutorials/retail_api_v2_write_user_events_java.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    The code samples for each of the Retail services are stored in different directories.
+
+1. Go to the code samples directory - our starting point to run more commands.
+    ```bash
+    cd java-retail/samples/snippets
+    ```
+
+## Write user events
+
+1. Before you start, build the Maven project and go to the code samples directory - our starting point to runcode samples:
```
Comment (DawidJeziorskiGrid) [inline tutorials/retail_api_v2_write_user_events_java.md:104]: Is:
our starting point to runcode samples:
maybe it should be:
our starting point to run code samples:

**Review by DawidJeziorskiGrid — APPROVED** [review]
Small errors from copied materials.

File: tutorials/retail_api_v2_purge_user_events_java.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    The code samples for each of the Retail services are stored in different directories.
+
+1. Go to the code samples directory - our starting point to run more commands.
+    ```bash
+    cd java-retail/samples/snippets
+    ```
+
+## Purge user events
+
+1. Before you start, build the Maven project and go to the code samples directory - our starting point to runcode samples:
```
[the remark this answers is not in the document]
Comment (pmichalskiGD) [reply tutorials/retail_api_v2_purge_user_events_java.md:105]: fixed

File: tutorials/retail_api_v2_write_user_events_java.md (hunk 2 above)
Comment (pmichalskiGD) [reply tutorials/retail_api_v2_write_user_events_java.md:104]: fixed

File: tutorials/retail_api_v2_rejoin_user_events_java.md (hunk 1 above)
Comment (pmichalskiGD) [reply tutorials/retail_api_v2_rejoin_user_events_java.md:109]: fixed

File: tutorials/retail_api_v2_purge_user_events_java.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+- `parent`—required field. The parent catalog name, such as `projects/<YOUR_PROJECT_NUMBER>/locations/global/catalogs/default_catalog`.
+- `filter`—required field. The filter string to specify the events to be deleted. An empty string filter is not allowed.
+  The eligible fields for filtering are:
+  - eventType: UserEvent.eventType string.
+  - eventTime: in ISO 8601 "zulu" format.
+  - visitorId: specifying this deletes all events associated with a visitor.
+  - userId: specifying this deletes all events associated with a user.
+- `force`—if force is set to `false`, the method returns the expected purge count without deleting any user events; if set to `true`, the user events are purged from the catalog.
+
+1. Open the`PurgeUserEventsRequest` example in the <walkthrough-editor-select-regex filePath="cloudshell_open/grs-retail/src/main/java/events/PurgeUserEvent.java" regex="# get purge user event request">`events/PurgeUserEvent.java`</walkthrough-editor-select-regex> file.
```
[the remark this answers is not in the document]
Comment (pmichalskiGD) [reply tutorials/retail_api_v2_purge_user_events_java.md]: done

**Review by t-karasova — APPROVED** [review]

**Pull request merged**